azurerm/resource_arm_automation_schedule.go

```diff
@@ -107,14 +108,94 @@ func resourceArmAutomationSchedule() *schema.Resource {
 				//todo figure out how to validate this properly
 			},
 
-			//todo missing properties: week_days, month_days, month_week_day from advanced automation section
+			"advanced_schedule": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"week_days": {
```

Comment:
>"week_days" [](start = 6, length = 11)

Since `week_days`, `month_days` and `monthly_occurrence` are mutually exclusive, why not declare `ConflictsWith` attribute in these three objects?